src/StockportWebapp/Views/Shared/CookieNotice.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿    <!-- Cookie Consent by FreePrivacyPolicy.com https://www.FreePrivacyPolicy.com -->
+    <script type="text/javascript" src="//www.freeprivacypolicy.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
+    <script type="text/javascript" charset="UTF-8">
```

Comment:
type attributes can be removed. default is text/js

Reply:
Done